### DISCUSSION TITLE
Restricting pay money

### DISCUSSION THREAD

**Discussion by sakulmore** [Zrips/CMI#7013] [Feature Requests]
Add to the config file maximum money player can send to another player.

For example: If there is a value like "maximumPay: 1500.0" <- A players can sends to another players maximum 1500$. Not more.

OR

Make it on a permission.

Examples:
When a group has permission "cmi.maxpay.1500" <- Then players, which are in certain group, can send to another player maximum 1500$.

When a group has permission "cmi.maxpay.500" <- Then players, which are in certain group, can send to another player maximum 500$.

etc...

**Comment by mrfloris** [comment]
I checked, i dont see a way to do this really. A probably not so safe method is to use a /pay command made with aliaseditor that checks against the $params to see if it is <= (or not) but smarty pants would perhaps still be able to bypass that by using /cmi pay or /cmi money, etc. So you'd have to fight with the permissions, run the custom commands asConsole! - but then check against their available balance and manage it.. feels like it's prone to mistakes. 

I see a fraction minimum amount is possible to be set, it would be the perfect place to set a default for max pay. where the default value is as much as the user has, and then server owner can set it to 1500, and override it with a maxpay perm node like you suggested so a donator can max send 4500
